snippets/python/Diamond_Square.md

```diff
@@ -0,0 +1,49 @@
+# Diamond Square
+
+## Description
+A Program to print the Diamond Sqaure Pattern, number of rows is given as input.
+
+## Code
+```python
+# The Diamond Pattern
+n=int(input("Enter number of Rows: "))
+
+# The Diamond Pattern
+n=int(input("Enter number of Rows: "))
+
+# The Top Pyramid
+for i in range(n):
+    space="  "*(n-i-1)
+    star=(2*i)*'* '+'*'
+    print(space+star)
+    
+# The bottom Pyramid
+for i in range(n):
+    space = ("    " if i!=0 else "  ")+"  "*(i-1)
+    star=star[:-4]
+    print(space+star)
+```
+
+## Input
+```
+Enter number of Rows: 5
+```
+
+## Output
+```
```

Comment:
Put the image of the output of the console instead of hardcoding the output

Reply:
Hey, to upload the image in the markdown file I need to put the image in markdown file too , shall I make another directory inside the python directory named as **Output Images** . Please tell me if there is any other way to include an image in the markdown file.

Reply:
You can create another folder under the Python folder named "Diamond Pattern"